config/locales/simple_form.nb.yml

```diff
@@ -44,6 +44,13 @@ nb:
         email_example: "epost@epost.no"
         session:
           heading_new: "Logg inn til din bruker"
+      space_contact:
```

Comment:
Har vi ikke alle disse i nb.yml under active record? Skal de her også? Ikke verdens undergang, men bare kjekt å vite!

Reply:
Dette er for simple form, må under "simple form" handleren. Kanskje mulig å gjøre noe med, men usikker på hvordan.